prow/cmd/generic-autobumper/helper_test.go

```diff
@@ -20,6 +20,52 @@ import (
 	"testing"
 )
 
+// TestImageAndTagFromName tests ImageFromName() and TagFromName().
+func TestImageAndTagFromName(t *testing.T) {
+	cases := []struct {
+		name          string
+		imageName     string
+		expectedImage string
+		expectedTag   string
+	}{
+		{
+			name:          "empty",
+			imageName:     "",
+			expectedImage: "",
+			expectedTag:   "",
+		},
+		{
+			name:          "basically works",
+			imageName:     "gcr.io/k8s-prow/test:tag1",
+			expectedImage: "gcr.io/k8s-prow/test",
+			expectedTag:   "tag1",
+		},
+		{
+			name:          "just a tag",
+			imageName:     "tag1",
```

Comment:
Nit: I think if there is no ":" in there, that it's actually an image (with `:latest` inferred, which I think is how Docker behaves typically). So this test case's name should be "just an image" instead, I think. Ah but you've already used the same test case name just below this one. Maybe just delete this test case, then?

Reply:
That's fair. Done!